lib/ec2/Commands.rb

```diff
@@ -0,0 +1,57 @@
+module Cumulus
+  module EC2
+    require "common/Commands"
+    class Commands < Cumulus::Common::Commands
+
+      def self.banner_message
+        "ec2: Manage EC2 instances and related configuration."
+      end
+
+      def self.usage_message
+        "Usage: cumulus ec2 [help|ebs|instances] [diff|list|migrate|sync] [asset]"
```

Comment:
usually, the usage message finds the module name by looking at the manager. since the manager can be either an EbsManager or InstanceManager, it confuses the thing.
one sec, I know a better way to do it. thanks for pointing this out.